### DISCUSSION TITLE
run solara apps using server ip address and custom port

### DISCUSSION THREAD

**Discussion by rh-afk** [widgetti/solara#248] [Q&A] [answered]
Hi all,

kindly please advise how to run solara apps using server ip address and with custom port since not working if i use localhost.

thanks

**Comment by rh-afk** [comment, marked as answer]
Hi all,

I just read solara help and we can specify host and port for run solara server. Perfect.
![image](https://github.com/widgetti/solara/assets/141318843/3af3987e-34aa-4640-8000-60dc62978479)

**Reply by maartenbreddels** [reply]
Good to hear you solved it yourself. Where would you expect this in the documentation?

**Reply by rh-afk** [reply]
hi @maartenbreddels , additional note to run apps   should be perfect if we can put in  [https://solara.dev/docs/deploying/self-hosted](url) below this sentences probably?:
![image](https://github.com/widgetti/solara/assets/141318843/8548e26d-d3cc-4552-862b-c2c929c76c4a)

thanks
